javascript.js

```diff
@@ -0,0 +1,99 @@
+const bands = [
+  {
+    image: 'images/band1.jpg',
+    name: 'Oblivion Reign',
+    whereFrom: 'Sweden',
+    description: 'With a crushing sonic assault, Oblivion Reign reigns supreme, channeling raw energy and unyielding brutality through their bone-shattering compositions.',
+  },
+
+  {
+    image: 'images/band2.jpg',
+    name: 'Serpent\'s Embrace',
+    whereFrom: 'Sweden',
+    description: 'Serpent\'s Embrace mesmerizes with their venomous melodies and haunting atmosphere, entwining listeners in a dark embrace of ethereal beauty and menacing power.',
+  },
+  {
+    image: 'images/band3.jpg',
+    name: 'Thunderstrike',
+    whereFrom: 'Latvia',
+    description: 'Thunderstrike commands the stage with electrifying energy, unleashing a storm of thunderous rhythms and soaring anthems that ignite the crowd in an explosive frenzy.',
+  },
+  {
+    image: 'images/band4.jpg',
+    name: 'Shadowstorm',
+    whereFrom: 'Lithuania',
+    description: 'From the depths of darkness, Shadowstorm emerges, weaving intricate harmonies and atmospheric layers to create a haunting sonic landscape that enchants and captivates.',
+  },
+  {
+    image: 'images/band5.jpg',
+    name: 'Crimson Dawn',
+    whereFrom: 'Estonia',
+    description: 'A symphony of darkness and passion, Crimson Dawn paints vivid musical portraits with their fiery melodies and poetic lyricism, leaving an indelible mark on the soul.',
+  },
+  {
+    image: 'images/band0.jpg',
+    name: 'Infernal Forge',
+    whereFrom: 'Finland',
+    description: 'Masters of relentless aggression, Infernal Forge forge their sound in the fires of hell, delivering a relentless onslaught of blistering riffs and demonic vocals.',
+  },
+];
+
+const burger = document.getElementById('burger');
+const burgerMenu = document.getElementById('burger-menu-page');
+const closeIcon = document.getElementById('fa-close');
+
+function exitFunction() {
+  burgerMenu.style.display = 'none';
+}
+
+closeIcon.addEventListener('touchstart', exitFunction);
+const seeMore = document.getElementById('see-more');
+
+function openMenu() {
+  burgerMenu.style.display = 'block';
+}
+
+burger.addEventListener('touchstart', openMenu);
```

Comment:
- [ ] As stated in the project requirement you are required to implement Mobile menu, When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
- The mobile menu should have a close (X) button that closes the menu. **But in the case of your Mobile menu when I clicked on the Hamburger menu nothing happens please check this [video](https://drive.google.com/file/d/1U9NmB6FdwG2NP6_lji7kMSex2fQklmDE/view?usp=sharing)** and Fix the issues and on the mobile menu I recommend you to include the interaction link between Home and About page for user interaction.